beetsplug/badfiles.py

```diff
@@ -82,6 +84,9 @@ def check_mp3val(self, path):
     def check_flac(self, path):
         return self.run_command(["flac", "-wst", path])
 
+    def check_wv(self, path):
```

Comment:
You could rename this function to `check_wavpack`. I understand that `.wv` is the file extension of wavpack but it is not too well known yet so it doesn't hurt to be a little more self-explanatory here. Thanks!